jina/drivers/multimodal.py

```diff
@@ -0,0 +1,114 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+from collections import defaultdict
```

Comment:
Import sequence can be standard

Reply:
what is the standard?

Reply:
https://stackoverflow.com/questions/22722976/import-order-coding-standard